lib/ansible/modules/network/aci/aci_rest.py

```diff
@@ -240,7 +237,7 @@ def main():
     aci = ACIModule(module)
 
     if method == 'get':
-        aci.request()
+        aci.request(path)
```

Comment:
There are some unit tests related to `module_utils.aci`, what do you think about adding unit tests related to this module ?